SparkPost/2web2ui
# AC-1339 - UI verifies and consumes promo codes via Billing API

AC-1339 - UI verifies and consumes promo codes via Billing API

### What Changed
 - replaced instances of GET and POST account/subscription/promo-codes/:code -> /billing/subscription/promo-codes/:code
 - updated tests - cypress and unit tests 
 - updated location of fixtures

### How To Test
 - Check onboarding page(/onboarding/plan) and plan change page(/account/billing/plan) for promocode field, if it works correctly.
~**- Note - the API ticket is in review so we might have to wait till it goes out to test this.**~

### To Do
